buildkite/bazelci.py

```diff
@@ -438,9 +438,17 @@
         "queue": "windows",
         "python": "python.exe",
     },
+    "rbe_macos": {
+        "name": "RBE (macOS, OpenJDK 8)",
+        "emoji-name": ":gcloud::darwin: (OpenJDK 8)",
+        "downstream-root": "/Users/buildkite/builds/${BUILDKITE_AGENT_NAME}/${BUILDKITE_ORGANIZATION_SLUG}-downstream-projects",
+        "publish_binary": [],
+        "queue": "macos",
+        "python": "python3.7",
+    },
```

Comment:
what does this do?

Reply:
1. Empty publish_binary -> Bazel binaries should NOT be uploaded to GCS since this feature is experimental, and the regular macos platform already publishes binaries.
2. Queue macos -> Required to get a MacOS worker
3. Python 3.7 -> The version of Python on the host machine that runs the bazelci script.